### PR TITLE
Changed sounds of levers and switches and knobs

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

Here we worked with tons of annoying templates and mind numbing work to bring you these sounds. As before, here we have the small changes:

- Improved the sound of the runway light thumps
- Made engine shutdown quieter in cockpit

Here we have the main, fat changes that are the best part of the PR:

- Changed the sound of the parking brake
- Changed the sound of the spoiler lever
- Changed the sound of the flap lever
- Changed the sound of the Engine master switches
- Changed the sound of the knobs such as the engine mode selector and ND Range knob
- Changed the sound of the RMP knobs
- Changed the sound of the QNH, and FCU altitude speed heading and VS speed knobs
- Changed the sound of the ADIRS knobs (separate to other knobs because IRL you would pull them before turning)
- Changed the sound of the cockpit door lock switch
- Changed the switch sound for seatbelt sign, no smoking sign, and that type of switch

Important: Button sounds and light switch sounds are coming in a separate PR - number #1877 - this will rework the buttons on the OVHD panel and the 8 light switches which will stay default for this PR

## References

These were recorded by komp in a cold and dark a320 cockpit, so everything is genuine and realistic - also confirmed by other pilots to sound realistic

## Additional context

templates really suck 

also the vertical speed knob should work now, if it doesn't then i dont know what to do

Discord username (if different from GitHub):

Boris#9134

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by hotshotp** [comment]
ALL ISSUES ARE NOW FIXED! Time for testing

**Comment by ImenesFBW** [comment]
Name: Imenes#8739
Date of testing: Current date 17/12/2020
Version of the sim: 1.11.7.0
PR Tested: #2466
Object of test: Testing the changed sounds of levers and switches and knobs
Testing technique: Try-out (multiple tests)

## Steps you've made to test the Pull Request:

 * Loaded C&D played with all the things
 * Did a short flight to check for bugs
 * Flipped switches while flying 

## Overall Rating: Good

#### Conclusions of the testing:
 * LGTM, another great PR from Boris can't wait to see it in the master.

**Comment by MapleNAV-TCASFAULT** [comment]
Category: QA Tester
Name: ChristopherXie#3904
Date of testing: Current date 12/16/2020
Version of the sim: 1.11.7.0
PR Tested: #2466 
Object of test: Sounds of knobs and switches
Testing technique: spawned on runway and did a touch and go
Steps you've made to test the Pull Request:
1. Checked all buttons that were mention (all ok)
2. Did a take off to test the altered ground roll
3. Did a landing to test the altered ground roll
4.
5.

Flight notes (If required):

Issues (If Any): None AFAIK
Overall Rating: Good
Conclusions of the testing: Great jobs on the knobs and switches Boris and Komp. Can't wait till all of the knobs and switches get replaced :D

File: A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml [hunk 1]
```diff
@@ -132,5 +132,197 @@
             <EMISSIVE_CODE_ACTIVE>(L:#PLANE_NAME#_AUTOPILOT_APPR_MODE, bool)</EMISSIVE_CODE_ACTIVE>
         </UseTemplate>
     </Template>
+    
+    	<Template Name="FBW_AUTOPILOT_Knob_SpeedMach_Template">
```
Comment (Benjozork) [inline A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Autopilot.xml:136]:
```suggestion
 <Template Name="FBW_AUTOPILOT_Knob_SpeedMach_Template">
```

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -154,13 +154,12 @@
...
 1. [CDU] F-PLN page visual rework - @Lollo999 (Lorenzo Pinna)
 1. [Sound] Improved ground roll - @hotshotp (Boris)
-1. [Sound] Added new turbulence sounds different in cabin and cockpit - @hotshotp (Boris)
+1. [Sound] Added new heavy turbulence sounds different in cabin and cockpit - @hotshotp (Boris)
 1. [CDU] Added ground speed mini - @MisterChocker (Leon)
 1. [MISC] Fix pedestal camera position triggering radio frequency display glitch - @pareil6 (pareil6)
 1. [FLIGHTMODEL] Corrected fuel tank capacities, including unusable fuel - @donstim - (donbikes#4084)
 1. [General] Added option to switch between KG and LBS - @MisterChocker (Leon)
-1. [CDU] Improved INIT-A PAGE transition to INIT-B or FUEL PRED PAGE - @MisterChocker (Leon)
-1. [CDU] Improved Options Page - @MisterChocker (Leon)
```
Comment (Benjozork) [inline .github/CHANGELOG.md:163]: Why was this removed ?

Comment (hotshotp) [reply .github/CHANGELOG.md:163]: I genuinely have no idea, could you fix that for me please

Comment (Benjozork) [reply .github/CHANGELOG.md]: Done.

**Review by Benjozork — APPROVED** [review]
LGTM
